slack.ts

```diff
@@ -0,0 +1,54 @@
+/** Error20:
+    - line 21: don't know how to declare call signature in tsc*/
+
+import * as querystring from "querystring";
+import fetch from "node-fetch";
+
+const tkn = process.env.SLACK_TOKEN;
+const domain = process.env.DOMAIN;
+
+async function sendMessage(message : string, config : string) : Promise<void> {
+    const msg = {
+        token: tkn,
+        channel: config,
+        as_user: false,
+        username: "HackGT",
+        text: message
+    };
+
+    const qs = querystring.stringify(msg);
+
+    const res = await
```

Comment:
It's OK to have this and the next line on the same line, this goes for all cases here where `await` and the next function are on different lines.